### PR TITLE
Peer Review

### PR DESCRIPTION
**Pull request by Lily-Lilian**

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                
+                </div>
+            </div>
+            <div class="nav__rightside">
+                <button class="button button--link">Login</button>
+                <button class="button">join us</button>
+            </div>
+        </nav>
+    <section class="middle__section">
+        <div class="middle__sectiontitle">Welcome
```
Comment (chrisostome01) [inline Index.html]: you can use kababa case here

File: Index.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        </div>
+        <div class="middle__sectiondescr">A Great Place to Receive Care</div>
```
Comment (chrisostome01) [inline Index.html]: use kabab case here too and use descriptive name

File: Index.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+        <div class="middle__sectionsubt">Get your best looking smile now!</div>
+        <div class="middle__sectionbtn">
+            <button class="button">Join Us</button>
+            <button class="button button--set">learn more</button>
+        </div>
+    </section>
+</header>
+<main class="main">
+<div class="intro">
+    <img class="main__introimg" src="./images/thumb-concept.png" alt="" >
```
Comment (chrisostome01) [inline Index.html]: also here, use kabab case

File: Index.html [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+</div>
+<div class="team-layout">
+    <div class="team__title">Team</div>
+    <div class="team__bigtitle">Our team</div>
+    <div class="team__text">Problems trying to resolve the conflict between 
+        the two major realms of Classical physics: Newtonian mechanics</div>
+       <div class="team__member">
+        <div class="team__card">
+          <div class="team__containerimg">
+            <img class="team__img" src="./images/user-cover-1.png" alt="">
```
Comment (chrisostome01) [inline Index.html]: always is good idea to add description to you alt artibute

File: Index.html [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./main.css">
+    <title>landing Page</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="nav">
```
Comment (chrisostome01) [inline Index.html]: since we have that tag in HTML it's better to use another descriptive name for it

**Review by chrisostome01 — CHANGES_REQUESTED** [review]
Well done @Lily-Lilian, I left some comments for you and you can also use `CRTL + SHFT + F`  to format your HTML doc

File: Index.html [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+    <div class="main__title">Meet Our Experts</div>
+    <div class="main__text">Problems trying to resolve the conflict between 
+        the two major realms of Classical physics: 
+        Newtonian mechanics
+    </div>
+    <div class="main__link">Learn More</div>
+</div>
+</div>
+<div class="team-layout">
+    <div class="team__title">Team</div>
```
Comment (ritha-kankindi) [inline Index.html]: Here the block is team-layout and it's elements must be team-layout__title, team-layout__(...).
 every element within this block must start by team-layout as it is a block name

File: Index.html [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+                    <div class="nav-item">Contact</div>
+                
+                </div>
+            </div>
+            <div class="nav__rightside">
+                <button class="button button--link">Login</button>
+                <button class="button">join us</button>
+            </div>
+        </nav>
+    <section class="middle__section">
```
Comment (ritha-kankindi) [inline Index.html]: Here, this element "middle__section" must inherit from "middle " block

**Review by chrisostome01 — APPROVED** [review]
LGTM

**Review by ritha-kankindi — APPROVED** [review]

File: Index.html [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+                <img src="./images/Vector1.svg" alt="facebook-icon" />
+                <img src="./images/Vector.svg" alt="twitter-icon" />
+                <img src="./images/Vector3.svg" alt="instragram-icon" />
+              </div>
+            </div>
+          </div>
+          <div class="team-layout__card">
+            <div class="team-layout__container-img">
+              <img
+                class="team__img"
```
Comment (karemarsy) [inline Index.html]: `team__img` is positioned outside the Block (team).

**Review by ritha-kankindi — APPROVED** [review]

**Review by karemarsy — APPROVED** [review]

**Pull request merged**